packages/@glimmerx/blueprint/dev/add-to-output-repos.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
I don't think we need this file, more modern ember-cli-update versions have a way to generate the output diff instead of forcing all blueprint publishers to save a repo.